<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>enter1Class</class>
 <widget class="QMainWindow" name="enter1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>都市圈最小代价生成树</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>131</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>1 南京</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2 镇江</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3 扬州</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4 淮安</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5 马鞍山</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6 滁州</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7 芜湖</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8 宣城</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9 溧阳</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10 金坛</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>431</width>
      <height>531</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>460</y>
      <width>231</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>导出最小生成树图</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>导出全联通图</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="another">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>590</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>点击切换</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
